implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -382,4 +385,20 @@ void toCounterTags() {
         assertThat(actual.get(0)).startsWith("my.counter,");
         assertThat(actual.get(0)).hasSize("my.counter,tag1=value1,dt.metrics.source=micrometer,tag2=value2 count,delta=0.0 1617796526714".length());
     }
+
+    @Test
+    void tooLongLine() {
```

Comment:
```suggestion
    void linesExceedingLengthLimitDiscardedGracefully() {
```
Or mention in a comment why this behavior is expected.